### ISSUE TITLE
Can't copy version number from About Zed

### ISSUE THREAD

**Issue by dandv** [zed-industries/zed#29361]
### Summary

Since the version number in About Zed is approaching the average human digit span, e.g. 0.183.10, it would be helpful to make it selectable.

### Description

Steps to reproduce:
1. Click the hamburger menu
2. Click About Zed
3. Attempt to select or copy the Zed version number (not the commit hash)

Expected Behavior: able to select the version number for pasting in issues etc.
Actual Behavior: unable to select the version number

![Image](https://github.com/user-attachments/assets/a9c74248-a68f-46db-9522-65558906ffdf)

### Zed Version and System Specs

0.183.10

**Comment by dandv** [comment]
Also can't copy error text:

![Image](https://github.com/user-attachments/assets/01ff17ba-5b6c-4ea2-9f31-6bb0454302af)

**Comment by OnorioCatenacci** [comment]
FWIW, I just tested this on the windows build (0.186.0) and it does work after a fashion.  You can't select the string but Windows allows you to copy a dialog to the clipboard so you can get the version string.  This is what I see:

> [Window Title]
> Info
> 
> [Main Instruction]
> Zed Dev 0.186.0 
> 
> [Content]
> 122af4fd53e57ff20aa96eb652ca90f9b108e6a8
> 
> [OK]

As I say, just some additional information.
